app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCStatus.scala

```diff
@@ -0,0 +1,295 @@
+package org.bitcoins.commons.jsonmodels.dlc
+
+import org.bitcoins.commons.jsonmodels.dlc.DLCMessage.{
+  DLCAccept,
+  DLCMutualCloseSig,
+  DLCOffer,
+  DLCSign
+}
+import org.bitcoins.core.protocol.transaction.Transaction
+import org.bitcoins.crypto.{SchnorrDigitalSignature, Sha256DigestBE}
+
+sealed trait DLCStatus {
+  def eventId: Sha256DigestBE
+  def isInitiator: Boolean
+  def offer: DLCOffer
+}
+
+sealed trait AcceptedDLCStatus extends DLCStatus {
+  def accept: DLCAccept
+}
+
+object DLCStatus {
+
+  def statusString(status: DLCStatus): String = {
+    status match {
+      case _: Offered         => "OFFERED"
+      case _: Accepted        => "ACCEPTED"
+      case _: Signed          => "SIGNED"
+      case _: Broadcasted     => "BROADCASTED"
+      case _: Confirmed       => "CONFIRMED"
+      case _: CloseOffered    => "CLOSE OFFERED"
+      case _: Closed          => "CLOSED"
+      case _: Claiming        => "CLAIMING"
+      case _: Claimed         => "CLAIMED"
+      case _: Penalized       => "PENALIZED"
+      case _: RemoteClaiming  => "REMOTE CLAIMING"
+      case _: RemoteClaimed   => "REMOTE CLAIMED"
+      case _: RemotePenalized => "REMOTE PENALIZED"
+      case _: Refunded        => "REFUNDED"
```

Comment:
Why not just give the `DLCStatus` trait a `statusString` function

Reply:
Also pref, title case over all caps

Reply:
My reasoning was that layer/responsibility-wise this seems like more of a front-end/client/user utility than something that belongs to the elements of the ADT.

Reply:
(this was originally something I wrote in the `DLCPane` but this seemed like a better place)